client/src/components/existing-request/names-gray-box.vue

```diff
@@ -1,41 +1,20 @@
 <template>
-  <v-row no-gutters class="bg-light-gray">
-    <v-col class="text-name pa-4">
```

Comment:
I couldn't use a v-col because it was full-width and I wanted the tooltip on only the text.